src/lib/cimb_translator/Cell.h

```diff
@@ -39,20 +39,20 @@ class Cell
 		const uchar* p = _img.ptr<uchar>(0) + (index * channels);
 
 		int increment = 1 + skip;
-		int toNextCol = channels * (_img.rows - _rows);
+		int toNextRow = channels * (_img.cols - _cols);
```

Comment:
The fun part of using square dimensions everywhere is unraveling all the places you mixed up X and Y :upside_down_face: